apps/server/src/modules/files-storage/uc/files-storage.uc.spec.ts

```diff
@@ -71,6 +71,24 @@ describe('FilesStorageUC', () => {
 		service = module.get(FilesStorageUC);
 		storageClient = module.get(S3ClientAdapter);
 		fileRecordRepo = module.get(FileRecordRepo);
+		fileRecords = [
+			fileRecordFactory.build({ parentId: userId, schoolId, name: 'text.txt' }),
+			fileRecordFactory.build({ parentId: userId, schoolId, name: 'text-two.txt' }),
+			fileRecordFactory.build({ parentId: userId, schoolId, name: 'text-tree.txt' }),
+		];
+		fileRecordRepo.findBySchoolIdAndParentId.mockResolvedValue([fileRecords, fileRecords.length]);
+
+		fileRecordRepo.save.mockImplementation((entity: FileRecord | FileRecord[]) => {
```

Comment:
I want to prefare that we switch fileRecordRepo.save to array only as input.

Reply:
Let's fix/discuss that in 'BaseRepo Ticket' ?

Reply:
Added to ticket.